### PR TITLE
quick fix of readme

### PR DESCRIPTION
**Pull request by cooktheryan**
Helpful for #424

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,13 +77,13 @@ rm -rf /var/lib/microshift && rm -r $HOME/.microshift
 In production environment MicroShift can be deployed as:
 
 1. Install via an RPM, utilizing a host-provided cri-o runtime and be lifecycle-managed by systemd
-2. [Install as a container via Podman, utilizing cri-o runtime and be lifecycle-managed by systemd](./docs/microshift-containerized/README.md)
+2. [Install as a container via Podman, utilizing cri-o runtime and be lifecycle-managed by systemd](https://microshift.io/docs/known-issues/known-issues/)
 
 For app developer deployments:
 
-1. [Run an all-in-one microshift deployment on which devs can test their applications locally](.docs/microshift-aio/README.md).  `microshift-aio` packages cri-o runtime and can be run and managed via podman and systemd
+1. [Run an all-in-one microshift deployment on which devs can test their applications locally](https://microshift.io/docs/known-issues/known-issues/).  `microshift-aio` packages cri-o runtime and can be run and managed via podman and systemd
```
Comment (sallyom) [inline README.md:84]: update to content/docs/deployment-modes/microshift-aio

File: README.md [hunk 2]
```diff
@@ -77,13 +77,13 @@ rm -rf /var/lib/microshift && rm -r $HOME/.microshift
 In production environment MicroShift can be deployed as:
 
 1. Install via an RPM, utilizing a host-provided cri-o runtime and be lifecycle-managed by systemd
-2. [Install as a container via Podman, utilizing cri-o runtime and be lifecycle-managed by systemd](./docs/microshift-containerized/README.md)
+2. [Install as a container via Podman, utilizing cri-o runtime and be lifecycle-managed by systemd](https://microshift.io/docs/known-issues/known-issues/)
```
Comment (sallyom) [inline README.md:80]: update the url for microshift-containerized

Comment (cooktheryan) [reply README.md:80]: bad paste...good catch

**Review by sallyom — APPROVED** [review]
/lgtm

**Pull request merged**